src/components/about/About.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import Icon from "../Icons";
+import IMG from '../../assets/img/card.png';
```

Comment:
its too general name, and if you will have one more img what to do then? 
rename it to `cardImg`, `aboutGif`... and the same for the rest of the components...